system/templates/chat/item-card.hbs

```diff
@@ -105,7 +105,12 @@
 
   {{#if data.item.system.damage.special}}
 	<div class="card-content">
-		Special Effect: {{{data.item.system.damage.special}}}
```

Comment:
The word "special effect" should be overwritten. That was the initial PR that was clearly incomplete.

Reply:
Note that the new change specifically uses `{{{data.itemSpecial.name}}}` and `{{{data.itemSpecial.system.description}}}` so that we explicitly use the "names in the game".

Reply:
Ah yes. Could you add "+ " before the `{{{data.itemSpecial.name}}}` so it shows that it should be added to the roll as well, that would be the format the monster attack description does in the PDFs/books?